packages/server/.storybook/preview.css

```diff
@@ -4,11 +4,6 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* html,
-body {
-  min-height: 100vh;
-} */
-
 @import 'https://fonts.googleapis.com/css?family=Roboto:400,500&display=block';
```

Comment:
i think these lines can also be removed now (but not 100% sure)

Reply:
No, this is the only way that storybook can have these fonts. The only other place they are added is to the app entry point, which storybook does not use.

Reply:
Oh gotcha